python/n.py

```diff
@@ -88,36 +88,37 @@ def display(self, start_line, start_col, end_line, end_col):
 class TypeCheckError:
 	def __init__(self, token_or_tree, message):
 		if type(token_or_tree) is not lark.Token and type(token_or_tree) is not lark.Tree:
-			exit()
-			#raise TypeError("token_or_tree should be a Lark Token or Tree.")
+			raise TypeError("token_or_tree should be a Lark Token or Tree.")
 		self.datum = token_or_tree
 		self.message = message
 
 	def display(self, display_type, file):
 		output = ""
+		color = Fore.RED
 		if display_type == "error":
 			output += f"{Fore.RED}{Style.BRIGHT}Error{Style.RESET_ALL}"
 		elif display_type == "warning":
 			output += f"{Fore.YELLOW}{Style.BRIGHT}Warning{Style.RESET_ALL}"
+			color = Fore.YELLOW
 		else:
-			exit()
-			#raise ValueError("%s is not a valid display type for TypeCheckError." % display_type)
+			raise ValueError("%s is not a valid display type for TypeCheckError." % display_type)
 		output += ": %s\n" % self.message
 		if type(self.datum) is lark.Token:
-			output += f"{Fore.CYAN}  --> {Fore.BLUE}run.n:{self.datum.line}:{self.datum.column}{Style.RESET_ALL}\n"
+			output += f"{Fore.CYAN} --> {Fore.BLUE}run.n:{self.datum.line}:{self.datum.column}{Style.RESET_ALL}\n"
 			output += file.display(
 				self.datum.line,
 				self.datum.column,
 				self.datum.end_line,
 				self.datum.end_column,
 			)
```

Comment:
Hmm, you probably meant to pass `color` in here as well?

Reply:
oh i just liked the alingment because it was better it that was removed

Reply:
ah ok